assets/home/components/HomeApp.tsx

```diff
@@ -230,6 +231,7 @@ class HomeApp extends React.Component<IProps, IState> {
 
     renderContent(children?: any): any {
         const {cards} = this.props;
+        const isWireSectionConfigured = Object.keys(this.props.userSections).find((key) => key === 'wire');
```

Comment:
Why iterate when you can check it directly? `this.props.userSections['wire'] != null`

Can you use a constant for `wire` string?